cms/check_rules_count_per_acc_st_exp_sexp_act

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
```

Comment:
The name of these script doesn't mean anything. Did you change the name from the one that is deleted? Can you give it a better name?